src/js/navButtons.js

```diff
@@ -96,7 +100,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             nextButtonId = fluid.allocateSimpleId(nextButton),
             showSelector = that.options.styles.show,
             isFirstPanel = currentPanelNum === that.options.panelStartNum,
-            nextLabel = isFirstPanel ? strings.start : (currentPanelNum === that.options.panelTotalNum ? strings.finish : strings.next);
+            nextLabel = isFirstPanel ? strings.start : (currentPanelNum === that.options.panelTotalNum ? strings.finish : strings.next),
+            nextTooltipContent = isFirstPanel ? strings.startTooltip : (currentPanelNum === that.options.panelTotalNum ? strings.finishTooltip : strings.nextTooltip);
```

Comment:
These two assignments are quite similar, can they be refactored in a way to share some of the logic?